CMakeLists.txt

```diff
@@ -206,6 +209,19 @@ set_target_properties(${_ALL_TARGETS} PROPERTIES
     CXX_STANDARD_REQUIRED ON
 )
 
+# Interprocedural optimization
+if(WarpX_IPO)
```

Comment:
Is it ok to move this logi into `cmake/WarpXFunctions.cmake`? Just to keep things concise here.

Reply:
Sure!

Reply:
It should be OK now...